modules/openid/cms_openid_module.e

```diff
@@ -203,7 +203,7 @@ feature -- Hooks
 				until
 					lnk2 /= Void
 				loop
-					if ic.item.location.same_string ("account/roc-logout") then
+					if ic.item.location.same_string ("account/roc-logout") or else ic.item.location.same_string ("basic_auth_logoff")  then
```

Comment:
This changes is not commented in the commit log.
Is it a bug correction, or recent addition due to cookies session  based auth?